gcs/src/main/scala/quasar/blobstore/gcs/GCSListService.scala

```diff
@@ -90,6 +121,9 @@ object GCSListings {
   implicit def gcsListingsEntityDecoder[F[_]: Sync]: EntityDecoder[F, GCSListings] = jsonOf[F, GCSListings]
   implicit def gcsListingsEntityEncoder[F[_]: Sync]: EntityEncoder[F, GCSListings] = jsonEncoderOf[F, GCSListings]
 
+  implicit val codecPageToken: CodecJson[PageToken] =
+    casecodec1(PageToken.apply, PageToken.unapply)("value")
```

Comment:
You may have tried to use this codec, but this definition does not match GCS response format. This codec specifies that it expects json with a `value` field, but once you have reached the path `nextPageToken ` then the response is just a string. If you specify something like this instead:

```suggestion
  implicit val codecPageToken: CodecJson[PageToken] =
    CodecJson.derived[String].xmap(PageToken(_))(_.value)
```

... (see next comment)